### PR TITLE
[bitnami/phpmyadmin] add servicemonitor

### PR DESCRIPTION
**Pull request by rechandler12**
**Description of the change**

Add ServiceMonitor to phpmyadmin

**Benefits**

**Possible drawbacks**

**Applicable issues**

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/phpmyadmin/Chart.yaml [hunk 1]
```diff
@@ -29,4 +29,4 @@ name: phpmyadmin
 sources:
   - https://github.com/bitnami/bitnami-docker-phpmyadmin
   - https://www.phpmyadmin.net/
-version: 7.0.1
+version: 7.0.2
```
Comment (marcosbc) [inline bitnami/phpmyadmin/Chart.yaml:32]: Due to the amount of changes, I would say this is a "minor" change (and not "patch"), i.e. it should be changed to "7.1.0".

**Review by marcosbc — CHANGES_REQUESTED** [review]
Thanks for the contribution! Adding a small request. The rest of the changes look good though.

File: bitnami/phpmyadmin/Chart.yaml (hunk 1 above)
Comment (rechandler12) [reply bitnami/phpmyadmin/Chart.yaml]: I've changed it. Can you resolve conflict?

**Review by marcosbc — APPROVED** [review]
LGTM, thanks!!!

**Pull request merged**